ts/components/conversation/Timestamp.tsx

```diff
@@ -39,14 +39,21 @@ export const Timestamp = (props: Props) => {
     return null;
   }
 
-  const momentValue = moment(timestamp);
-  // this is a hack to make the date string shorter, looks like moment does not have a localized way of doing this for now.
+  let title = '';
+  let dateString = '';
 
-  const dateString = momentFromNow
-    ? momentValue.fromNow().replace('minutes', 'mins').replace('minute', 'min')
-    : momentValue.format('lll');
+  // NOTE some existing groups might not have a joinedAtSeconds and we use a fallback value of 1 in order to poll and show up in the conversations list
+  if (timestamp !== 1) {
```

Comment:
should maybe be a named constant that we import on both files.
Just because when we get to remove it we will have to search for !== 1 which might be a pain

Reply:
Fixed